test/e2e/common/util.go

```diff
@@ -889,6 +889,67 @@ func IsVolumePublished(uuid string) bool {
 	return true
 }
 
+func CheckNoVolumeResources() (bool, bool) {
+	logf.Log.Info("CheckNoVolumeResources")
+	foundResources := false
+	success := true
+
+	pvcs, err := gTestEnv.KubeInt.CoreV1().PersistentVolumeClaims("default").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Error(err, "CheckNoVolumeResources: list PVCs failed.")
+		success = false
+	} else if pvcs != nil && len(pvcs.Items) != 0 {
+		logf.Log.Info("CheckNoVolumeResources: found PersistentVolumeClaims",
+			"PersistentVolumeClaims", pvcs.Items)
+		foundResources = true
+	}
+
+	pvs, err := gTestEnv.KubeInt.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Error(err, "CheckNoVolumeResources: list PVs failed.")
+		success = false
+	} else if pvs != nil && len(pvs.Items) != 0 {
+		logf.Log.Info("CheckNoVolumeResources: found PersistentVolumes",
+			"PersistentVolumes", pvs.Items)
+		foundResources = true
+	}
+
+	msvGVR := schema.GroupVersionResource{
+		Group:    "openebs.io",
+		Version:  "v1alpha1",
+		Resource: "mayastorvolumes",
+	}
+
+	msvs, err := gTestEnv.DynamicClient.Resource(msvGVR).Namespace("mayastor").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		logf.Log.Info("CheckNoVolumeResources: list MSVs failed.", "Error", err)
+		success = false
+	} else if msvs != nil && len(msvs.Items) != 0 {
+		logf.Log.Info("CheckNoVolumeResources: found MayastorVolumes",
+			"MayastorVolumes", msvs.Items)
+		foundResources = true
+	}
+
+	return foundResources, success
```

Comment:
Looks like foundResources acts as !success and vice versa. Maybe we can use one flag ?

Reply:
success => I was able to query the system
found_resources => found mayastor resources like volumes ( none expected by caller)

Reply:
Yes, indeed.

Reply:
The objective is to give more accurate report in the logs of a test run.